requirements/common.txt

```diff
@@ -1,29 +1,29 @@
 asgi-redis==1.4.3
-boto3==1.13.16
-botocore==1.16.16
+boto3==1.16.28
+botocore==1.19.28
 vine==1.3.0
 celery[sqs]==4.3.0
-commonmark==0.5.4
-django==2.2.8
+commonmark==0.9.1
+django==2.2.13
 django-allauth==0.43.0
-django-filter==2.1.0
-django-import-export==2.0.2
+django-filter==2.4.0
+django-import-export==2.4.0
 djangorestframework==3.10.3
 djangorestframework-expiring-authtoken==0.1.4
-django-cors-headers==1.3.1
-django-rest-auth[with_social]==0.9.2 
-django-ses==0.8.5
-docker-compose==1.21.0
+django-cors-headers==3.5.0
+django-rest-auth[with_social]==0.9.5
+django-ses==1.0.3
+docker-compose==1.27.4
 drfdocs==0.0.11
 drf-yasg==1.17.0
-kubernetes==10.0.1
-moto==1.3.8
-pika==0.10.0
-pickleshare==0.7.4
-Pillow==6.2.0
+kubernetes==12.0.1
+moto==1.3.14
```

Comment:
You said in your previous PR that `moto` has some issue, right?

Reply:
we are not updating moto to 1.3.15 that is the latest version which has isses